src/store/src/Bridge/Meilisearch/Store.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Store\Bridge\Meilisearch;
+
+use Symfony\AI\Platform\Vector\NullVector;
+use Symfony\AI\Platform\Vector\Vector;
+use Symfony\AI\Store\Document\Metadata;
+use Symfony\AI\Store\Document\VectorDocument;
+use Symfony\AI\Store\Exception\InvalidArgumentException;
+use Symfony\AI\Store\InitializableStoreInterface;
+use Symfony\AI\Store\VectorStoreInterface;
+use Symfony\Component\Uid\Uuid;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final readonly class Store implements InitializableStoreInterface, VectorStoreInterface
+{
+    /**
+     * @param string $embedder        The name of the embedder where vectors are stored
+     * @param string $vectorFieldName The name of the field int the index that contains the vector
+     */
+    public function __construct(
+        private HttpClientInterface $httpClient,
+        private string $endpointUrl,
+        #[\SensitiveParameter] private string $apiKey,
+        private string $indexName,
+        private string $embedder = 'default',
+        private string $vectorFieldName = '_vectors',
+    ) {
+    }
+
+    public function add(VectorDocument ...$documents): void
+    {
+        $response = $this->request('PUT', 'documents', array_map(
+            fn (VectorDocument $document): array => [$this, 'convertToIndexableArray'], $documents)
+        );
```

Comment:
isn't it the same like?´
```suggestion
        $response = $this->request('PUT', 'documents', array_map($this->convertToIndexableArray(...), documents));
```